test/spec/algoliasearch.helper/client.js

```diff
@@ -58,11 +64,17 @@ test('clearCache gets called if exists', function() {
 
 test('setting the agent once', function() {
   var client = algoliasearch('what', 'wait', {});
+  client._ua || Object.defineProperty(client, '_ua', {
+    get() {
+      return client.transporter.userAgent.value;
+    }
+  });
+
   var originalUA = client._ua;
   algoliaSearchHelper(client, 'IndexName', {});
   algoliaSearchHelper(client, 'IndexName2', {});
 
-  expect(client._ua).toBe(originalUA + ';JS Helper (' + version + ')');
+  expect(client._ua).toBe(originalUA + '; JS Helper (' + version + ')');
```

Comment:
Did you check with the Data team that these extra spaces don't cause trouble with their aggregation?

Reply:
No problem on that. It's already like this since version `3.33`. We are updating now, because previously we were testing against `3.32`: https://github.com/algolia/algoliasearch-helper-js/pull/756/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L43.

Now we are testing against `3.35.1`: https://github.com/algolia/algoliasearch-helper-js/pull/756/files#diff-1d37e48f9ceff6d8030570cd36286a61R67